examples/Tutorial.ipynb

```diff
@@ -0,0 +1,1262 @@
+      "       'EXAMPLE_68', 'EXAMPLE_82', 'EXAMPLE_110', 'EXAMPLE_12',\n",
+      "       'EXAMPLE_36', 'EXAMPLE_9', 'EXAMPLE_19', 'EXAMPLE_56',\n",
+      "       'EXAMPLE_104', 'EXAMPLE_69', 'EXAMPLE_55', 'EXAMPLE_132',\n",
+      "       'EXAMPLE_29', 'EXAMPLE_127', 'EXAMPLE_26', 'EXAMPLE_128',\n",
+      "       'EXAMPLE_131', 'EXAMPLE_145', 'EXAMPLE_108', 'EXAMPLE_143',\n",
+      "       'EXAMPLE_45', 'EXAMPLE_30', 'EXAMPLE_22', 'EXAMPLE_15',\n",
+      "       'EXAMPLE_65', 'EXAMPLE_11', 'EXAMPLE_42', 'EXAMPLE_146',\n",
+      "       'EXAMPLE_51', 'EXAMPLE_27', 'EXAMPLE_4', 'EXAMPLE_32',\n",
+      "       'EXAMPLE_142', 'EXAMPLE_85', 'EXAMPLE_86', 'EXAMPLE_16',\n",
+      "       'EXAMPLE_10', 'EXAMPLE_81', 'EXAMPLE_133', 'EXAMPLE_137',\n",
+      "       'EXAMPLE_75', 'EXAMPLE_109'], dtype='<U11'), 'labels': array(['versicolor', 'setosa', 'virginica', 'versicolor', 'versicolor',\n",
+      "       'setosa', 'versicolor', 'virginica', 'versicolor', 'versicolor',\n",
+      "       'virginica', 'setosa', 'setosa', 'setosa', 'setosa', 'versicolor',\n",
+      "       'virginica', 'versicolor', 'versicolor', 'virginica', 'setosa',\n",
+      "       'virginica', 'setosa', 'virginica', 'virginica', 'virginica',\n",
+      "       'virginica', 'virginica', 'setosa', 'setosa', 'setosa', 'setosa',\n",
+      "       'versicolor', 'setosa', 'setosa', 'virginica', 'versicolor',\n",
+      "       'setosa', 'setosa', 'setosa', 'virginica', 'versicolor',\n",
+      "       'versicolor', 'setosa', 'setosa', 'versicolor', 'virginica',\n",
+      "       'virginica', 'versicolor', 'virginica'], dtype='<U10'), 'features': <50x4 sparse matrix of type '<class 'numpy.float64'>'\n",
+      "\twith 200 stored elements in Compressed Sparse Row format>, 'vectorizer': DictVectorizer(dtype=<class 'numpy.float64'>, separator='=', sort=True,\n",
+      "        sparse=True)}\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(test_examples)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 25,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Training data will be shuffled to randomize grid search folds.  Shuffling may yield different results compared to scikit-learn.\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Best objective : 0.93997113997114\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/srv/conda/envs/notebook/lib/python3.7/site-packages/sklearn/svm/base.py:922: ConvergenceWarning: Liblinear failed to converge, increase the number of iterations.\n",
+      "  \"the number of iterations.\", ConvergenceWarning)\n"
+     ]
+    }
+   ],
+   "source": [
+    "from skll import Learner\n",
+    "\n",
+    "learner = Learner('LinearSVC')\n",
+    "best_obj, grid_search_results = learner.train(train_examples, grid_objective='f1_score_micro')\n",
```

Comment:
It seems like this isn't converging. Can you increase the number of iterations, or are you concerned it will take too long? If not, maybe suppress the warning?

Reply:
I think this should get fixed by increasing `max_iter` which I might already have done in a recently merged PR. May be merge in `master` and see if it goes away?